src/formBuilder/Card.test.js

```diff
@@ -222,5 +226,28 @@ describe('Card', () => {
 
     const inputTypeLabel = getHeadingText(wrapper, 3);
     expect(inputTypeLabel).toContain('Custom Input Type');
+
+    const placeHolderLabel = getHeadingText(wrapper, 3);
+    expect(placeHolderLabel).toContain('Custom Input Type');
```

Comment:
I think this assertion is incorrect.  I think you should be looking at the 4th element and it should be the string "Custom Placeholder"